content/en/docs/reference/kubectl/docker-cli-to-kubectl.md

```diff
@@ -7,7 +7,7 @@ reviewers:
 ---
 
 <!-- overview -->
-You can use the Kubernetes command line tool `kubectl` to interact with the API Server. Using kubectl is straightforward if you are familiar with the Docker command line tool. However, there are a few differences between the Docker commands and the kubectl commands. The following sections show a docker sub-command and describe the equivalent `kubectl` command.
+You can use the Kubernetes command line tool `kubectl` to interact with the API Server. Using kubectl is straightforward if you are familiar with the docker command line tool. However, there are a few differences between the Docker commands and the kubectl commands. The following sections show a docker sub-command and describe the equivalent `kubectl` command.
```

Comment:
I feel like it should be Docker itself rather than changing it to docker

Reply:
As w wrote before @tengqm :
- We refer to the Docker technology in general as "Docker", and the docker command line as "docker".
- So which one would be best?

Reply:
@Aut0R3V could you please give you opinion?
Should be changed to **Docker**?

Reply:
Maybe it can be 'docker' command line tool. I got confused there thinking you were speaking of Docker in general.

Reply:
Hi all!

As this is a third party piece of technology, and as such we do not own the copyright to it.

According to [Docker's brand guidelines](https://www.docker.com/legal/trademark-guidelines), Docker should be capitalized in all instances.

As such, @Aut0R3V, please capitalize both instances of Docker and I can approve this for you.

Reply:
@celestehorgan capitalized. Please approve this. Thanks